### PR TITLE
Update to Argotronic.ArgusMonitor 6.2.3.2665

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
Resolves #103354 

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103380)

### CODE REVIEW

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2665
```
Comment (mdanish-kh) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 6.2.03
```

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2665
```
Comment (mdanish-kh) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.yaml:4]:
```suggestion
PackageVersion: 6.2.03
```

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Argotronic.ArgusMonitor
+PackageVersion: 6.2.3.2665
```
Comment (mdanish-kh) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.installer.yaml:4]:
```suggestion
PackageVersion: 6.2.03
```

File: manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- ProductCode: ArgusMonitor
```
Comment (mdanish-kh) [inline manifests/a/Argotronic/ArgusMonitor/6.2.3.2665/Argotronic.ArgusMonitor.installer.yaml:10]:
```suggestion
- ProductCode: ArgusMonitor
  DisplayVersion: 6.2.3.2665
```

**Comment by BrandonWanHuanSheng** [comment]
I finally confirmed.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Marketing version on the publisher's website is `6.2.03`. You should update the PackageVersion to 6.2.03 and add DisplayVersion 6.2.3.2665 for WinGet to match upgrades

**Comment by mdanish-kh** [comment]
[Policy] reset feedback

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
You would need to delete `6.2.3.2665` folder

**Review by mdanish-kh — APPROVED** [review]

**Comment by BrandonWanHuanSheng** [comment]
I can see that file.

**Comment by mdanish-kh** [comment]
Not necessarily the number 3, but only one manifest folder. `.yaml` files can be more than 3 depending on if we have more locale files for the package

**Pull request merged**